reference/api/settings.mdx

```diff
@@ -1910,3 +1911,189 @@ Reset an index's typo tolerance settings to their [default value](#typo-toleranc
 ```
 
 You can use the returned `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
+
+## Embedders (experimental)
+
+Embedders translate documents and queries into vector embeddings. You must configure at least one embedder to use AI-powered search.
+
+### Embedders object
+
+The embedders object may contain an arbitrary number of embedder objects:
```

Comment:
should we specify "with arbitrary names"?

BTW the number is not arbitrary, it must be <256. Up to 256 embedders